politico/api/v1/party/modal.py

```diff
@@ -0,0 +1,120 @@
+"""
+    contains the class for party modal
+"""
+
+class Party:
+    def __init__(self, id, name, hqAddress, logoUrl):
+        self.id = id
+        self.name = name
+        self.hqAddress = hqAddress
```

Comment:
Use pep8 standard for naming variables all through.
```suggestion
        self.hq_address = hqAddress
```